### DISCUSSION TITLE
Root URL forbidden

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [YOURLS/YOURLS#3638] [Q&A] [answered]
Is it possible to set things up so that when someone browses to the root url, they get redirected to another page e.g. google.com.

**Comment by dgw** [comment, marked as answer]
Sure. Create an `index.html` or `index.php` file that displays/does what you want.

YOURLS doesn't ship with any default behavior at `/`; this is an intentional design decision.

**Reply by ZuluPilgrim** [reply]
Ok, thanks
